client.go

```diff
@@ -57,7 +57,7 @@ type Deployer interface {
 // Updater of a deployed service function with new image.
 type Updater interface {
 	// Deploy a service function of given name, using given backing image.
-	Update(name, image string) error
+	Update(name, language, image string) error
```

Comment:
I'm not seeing where the language is being used for update?  Is this an early addition that should be part of replacing the KN binary dependency later?

Reply:
Update calls `Build()`.

Reply:
Ah, it's down in build.  Resolving.

Reply:
You are right, it's unused.